source/grid/grid_out.cc

```diff
@@ -3718,9 +3718,9 @@ GridOut::write_mesh_per_processor_as_vtu(
                unsigned int,
                std::string,
                DataComponentInterpretation::DataComponentInterpretation>>
-                        vector_data_ranges;
-  DataOutBase::VtkFlags flags;
-  DataOutBase::write_vtu(patches, data_names, vector_data_ranges, flags, out);
+    vector_data_ranges;
+  DataOutBase::write_vtu(
+    patches, data_names, vector_data_ranges, vtu_flags, out);
```

Comment:
While working on adapting the compression_level for the failing tests, I guess I found a bug in the code. Despite being able to set a `compression_level` using the `GridOutFlags::Vtu`, those were ignored here and instead `Vtkflags` with default settings have been passed to the `write_vtu` method

Reply:
This is too funny. Wait, what day is today?😃

Reply:
Ha, that's wild. I guess this comes from a time when we did not actually have separate `VtuFlags`.

I'll accept this patch as is, but would you mind adding an entry to `doc/news/changes/minor` that describes this particular problem and fix?